Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM rocker/geospatial:4.1.0
 
 # Set up main directory
-RUN mkdir --parents /home/fbf/
+RUN mkdir --parents /home/fbf/forecast
```

Comment:
i am not sure why you preferer to create forecast directory here

Reply:
It's needed for Option 1 as described [here](https://github.com/rodekruis/Typhoon-Impact-based-forecasting-model/pull/39#discussion_r679966025)